examples/stable-diffusion/training/train_dreambooth.py

```diff
@@ -0,0 +1,1365 @@
+#!/usr/bin/env python
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+Training script for DreamBooth to Text-to-Image Diffusion Models
+Adapted from the following source:
+https://github.com/huggingface/peft/blob/608a90ded9985ee1c5912d738082bb1fd618902b/examples/stable_diffusion/train_dreambooth.py
```

Comment:
This is from the PEFT repo? But this is not the LoRA version no?
Why not using https://github.com/huggingface/diffusers/blob/v0.26.3/examples/dreambooth/train_dreambooth.py?

Reply:
this is from peft repo. train_dreambooth only provide full finetune. but the train_dreambooth in peft provide full/lokr/lora/loha and I also extend oft in it.